### ISSUE TITLE
Add FSPS model based on the MIST isochrone

### ISSUE THREAD

**Issue by dr-guangtou** [moustakas/impro#7]
FSPS has updated to the new MIST isochrone: http://waps.cfa.harvard.edu/MIST/ 

It would be great to have it in iSEDFit! Let me know if there is anything I can help.

**Comment by moustakas** [comment]
@dr-guangtou 

I'm just taking a look at this--it's been a long time since I've added models to `iSEDfit`. One thing I realized is I need a grid of SSPs with these isochrones.  To match my existing models, it would be useful if these were generated with both the `MILES` and `BaSeL` stellar libraries and for the `Chabrier`, `Kroupa`, and `Salpeter` IMFs.  

The output files (for each available metallicity) should be in the `FSPS` `.spec` format with the filename, e.g., `SSP_Padova_CKC14z_Kroupa_Z0.0003.out.spec`.

Are these spectra something you would be able to generate yourself using either `FSPS` or `python-fsps`?

Some more notes on metallicity:

It looks like the `MIST` isochrones exist for 
```
['m2.50','m2.00','m1.75','m1.50','m1.25','m1.00','m0.75',
  'm0.50','m0.25','p0.00','p0.25','p0.50']                                                      
```
or
```
Z = 10^[-2.5,-2,-1.75,-1.25,-1,-0.75,-0.5,-0.25,0.0,0.25,0.5]*0.019
  6.0083272e-05   0.00018999999   0.00033787309    0.0010684485    
  0.0018999999    0.0033787307 0.0060083275     0.010684485     0.018999999     
  0.033787306     0.060083274
```
assuming `Zsun=0.019`.  

For `MILES` my notes say that the only metallicities available are `Z=0.0008, 0.0031, 0.0096, 0.0190, 0.0300` so we'll need to be a little careful.  The [manual](https://github.com/cconroy20/fsps/blob/master/doc/MANUAL.pdf) may include some useful information or we can reach out to Charlie or Ben.

**Comment by dr-guangtou** [comment]
Yes, I can generate the SSP spectra using `fsps`.  It should be very straightforward.  

The `MIST` isochrone does allow more metallicity, but I am not sure all of them are "safe".  I can take a look of the `MIST` paper and check with Ben on this.  

If it is not too much trouble, it would be interesting to include the new Parsec isochrones too.  I can provide all the `.spec` files.

**Comment by dr-guangtou** [comment]
And, BTW, `MIST` uses a updated `Zsun=0.0142` based on Asplund et al. 2009; `Parsec` assumes `Zsun=0.01524`.  

These can be found at line 166 of `sps_vars.f90`.

**Comment by dr-guangtou** [comment]
I just asked Ben Johnson.  Apparently for SSP with metallicity lower than the spectral library limit, you will basically get the closest metallicity (lowest Z) one.  

In this case, I think we should ignore all SSPs with Z lower than the lower boundary of the spectral library (when using MILES).
